fix various delegate docs (#635)
harness/developer-hub: docs/platform/2_Delegates/delegate-reference/common-delegate-profile-scripts.md

> You can add any commands supported on the host/container/pod running the delegate. Linux shell commands are most common. If `kubectl`, Helm, or Docker is running on the host/container/pod where you install the delegate, then you can use their commands. Kubernetes and Docker delegates include Helm.

Reviewer: ```suggestion
You can add any command that the host, container, or pod running the delegate supports. Linux shell commands are most common. If `kubectl`, Helm, or Docker is running on the host, container, or pod where you install the delegate, you can use those commands. Kubernetes and Docker delegates include Helm.
```

> The base image for the delegate is Ubuntu 18.04 or later. This means you can use any default Ubuntu package in delegate script.

Reviewer: ```suggestion
The base image for the delegate uses Ubuntu 18.04 or later. This means you can use any default Ubuntu package in delegate script.
```

### Pip

> Ensure that you run `microdnf update` before running any `microdnf` commands.

Reviewer: ```suggestion
Run `microdnf update` before you run `microdnf` commands.
```

```
microdnf update  
# Install pip  
microdnf -y install python3-pip  
# Check pip install  
pip -v
```

### Unzip

> Ensure that you run `microdnf update` before running any `microdnf` commands.

Reviewer: ```suggestion
Run `microdnf update` before you run `microdnf` commands.
```
